### PR TITLE
YLP-1699 Rework TeamUser structure so that it becomes understandable

### PR DESCRIPTION
**Pull request by TimCanu**
- Deleted TeamUser structure (cannot be more understandable than that)

### CODE REVIEW

**Comment by TimCanu** [comment]
Successful yourloops build using BETA-22: https://jenkins.ci.diabeloop.eu/blue/organizations/jenkins/mdblp%2Fyourloops/detail/engineering%2FYLP-1699-teamUser-rework/2/pipeline/

File: packages/yourloops/lib/team/utils.tsx [hunk 1]
```diff
@@ -77,92 +72,55 @@ export default class TeamUtils {
...
-
-  static async loadTeams(user: User): Promise<Team[]> {
-    const users = new Map<string, TeamUser>()
-    const apiTeams = await TeamApi.getTeams()
-
-    log.debug('loadTeams', { nTeams: apiTeams.length })
+  static async loadTeams(userId: string, invitations: INotification[]): Promise<Team[]> {
+    const iTeams = await TeamApi.getTeams()
 
     const privateTeam: Team = {
```
Comment (fatb38) [inline packages/yourloops/lib/team/utils.tsx:96]: Why do we create a privateTeam ?

File: packages/yourloops/lib/team/index.ts [hunk 2]
```diff
@@ -45,7 +44,6 @@ import { getDisplayTeamCode } from './utils'
 
 export {
```
Comment (fatb38) [inline packages/yourloops/lib/team/index.ts:45]: Can you remove this (useless) file ?

File: packages/yourloops/lib/team/utils.tsx (hunk 1 above)
Comment (TimCanu) [reply packages/yourloops/lib/team/utils.tsx:96]: From what I know, this is useful for directshares

Comment (fatb38) [reply packages/yourloops/lib/team/utils.tsx:96]: ok !

**Review by fatb38 — APPROVED** [review]

**Pull request merged**